src/applications/personalization/profile360/tests/components/PaymentInformationEditModalError.unit.spec.jsx

```diff
@@ -66,6 +66,68 @@ describe('<PaymentInformationEditModalError />', () => {
       },
     ],
   };
+  const badHomeAreaCodeError = {
```

Comment:
The `key` and `text` values in these new error messages have been pulled from Sentry. The text values are passed through from the BGS service which is a service that the EVSS PPIU service hits. We do not have access to the full list of messages we might get back so we're doing the best that we can on short notice. Eventually EVSS PPIU will adjust how they interact with BGS and we should no longer get errors related to bad contact info at some point in the future (roughly six weeks from now)